src/sections/grid.ts

```diff
@@ -0,0 +1,8 @@
+import { MethodFn } from "../config";
+import { newMethodsForProp, newParamMethod } from "../methods";
+
+export const grid: MethodFn = () => [
+  newParamMethod("gtc", "gridTemplateColumns"),
+  newParamMethod("gtr", "gridTemplateRows"),
+  newParamMethod("gap", "gap"),
```

Comment:
Wdyt about making `newIncrementMethod` here so that we get `gap1`, `gap2` etc?

Reply:
Good idea! I've just included that!